From d88b86289aaed5713a44f204a9c07a1e192fd798 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 20 Sep 2012 14:56:00 +0800
Subject: [PATCH 341/502] ENGR00225144 lights: Add lights to Makefile

Add lights HAL to top Makefile.
Remove the deprecated PRELINK define.

Change-Id: I3ef0a484ada836c6e47a3ae456d86af76383c585
Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 Android.mk        | 4 ++--
 lights/Android.mk | 1 -
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/Android.mk b/Android.mk
index 03db119..ea6c5a5 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,6 +1,6 @@
-common_imx_dirs := libsensors alsa libgps
+common_imx_dirs := libsensors alsa libgps lights
 mx5x_dirs := $(common_imx_dirs) mx5x/libcopybit mx5x/libgralloc  mx5x/hwcomposer mx5x/libcamera
-mx6_dirs := $(common_imx_dirs) mx6/libgralloc_wrapper mx6/hwcomposer mx6/libcamera ion
+mx6_dirs := $(common_imx_dirs) mx6/libgralloc_wrapper mx6/hwcomposer mx6/libcamera
 
 ifeq ($(TARGET_BOARD_PLATFORM),imx6)
   include $(call all-named-subdir-makefiles,$(mx6_dirs))
diff --git a/lights/Android.mk b/lights/Android.mk
index d01e5c8..cc2573a 100644
--- a/lights/Android.mk
+++ b/lights/Android.mk
@@ -17,7 +17,6 @@ LOCAL_PATH := $(call my-dir)
 
 # HAL module implemenation, not prelinked and stored in hw/
 include $(CLEAR_VARS)
-LOCAL_PRELINK_MODULE := true
 LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
 LOCAL_SHARED_LIBRARIES := liblog libcutils
 LOCAL_SRC_FILES := fsl_lights.c
-- 
1.8.0

